apps/library/src/crt0.s

```diff
@@ -0,0 +1,48 @@
+_start:
+  .global _start
+  .section .vectors, "ax"
+  .option norvc;
+  .org 0x00
+  nop
+  nop
+  nop
+  nop
+  nop
+  .org 0x14
+  jal x0, reset_handler
+
+reset_handler:
+  mv  x1, x0
+  li  x2, 0x00001E00
```

Comment:
Let's try and set the SP not in a Hard coded way in the crt0.
We can Include a "asap_defs.h" files - somehow connect this also to the linker?